h2o-core/src/main/java/water/api/schemas3/ModelSchemaV3.java

```diff
@@ -34,6 +34,9 @@
   @API(help="The build parameters for the model (e.g. K for KMeans).", direction=API.Direction.OUTPUT)
   public PS parameters;
 
+  @API(help="The input parameters for the model (what was actually used, when parameter was not specified/was specified as AUTO).", direction=API.Direction.OUTPUT)
+  private PS input_parameters;
+  
```

Comment:
is it still needed now that `ModelParameterSchemaV3` has an `input_value` field?

Reply:
no no, not anymore - I just wanted to make work input_values first and after that get rid of effective parameters

Reply:
@koniecsveta replying to https://github.com/h2oai/h2o-3/pull/4361#issuecomment-643996065

I just want to ensure we don't expose twice the same thing or worse, make it confusing.
I just realize that you made it private, so it is probably not exposed in the REST API: can you please remove the `@API` annotation then?
It should just be needed for this call below, right?
```
      ModelParametersSchemaV3.writeParametersJSON(ab, parameters, input_parameters, defaults ,"parameters");
```